packages/unlock-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unlock-protocol/unlock-js",
-  "version": "0.28.2",
```

Comment:
Since this does not change the API I would only do a patch

Reply:
It does because it changes the export name.